backend/src/main/java/ca/mcgill/purposeful/controller/IdeaController.java

```diff
@@ -49,12 +61,49 @@ public ResponseEntity<IdeaDTO> getIdeaById(@PathVariable String id) {
   @PreAuthorize("hasAnyAuthority('User', 'Moderator', 'Owner')")
   public ResponseEntity<List<IdeaDTO>> filterIdeas(@RequestBody SearchFilterDTO searchFilterDTO) {
     return ResponseEntity.status(HttpStatus.OK)
-        .body(
-            IdeaDTO.convertToDto(
-                ideaService.getIdeasByAllCriteria(
-                    searchFilterDTO.getDomains(),
-                    searchFilterDTO.getTopics(),
-                    searchFilterDTO.getTechnologies())));
+        .body(IdeaDTO.convertToDto(ideaService.getIdeasByAllCriteria(searchFilterDTO.getDomains(),
+            searchFilterDTO.getTopics(), searchFilterDTO.getTechnologies())));
+  }
+
+  /**
+   * This method creates an idea
+   *
+   * @return created idea
+   * @throws Exception
+   * @author Adam Kazma
+   */
+  @PostMapping(value = {"/create", "/create/"})
+  @PreAuthorize("hasAuthority('User')")
+  public IdeaDTO createIdea(@RequestBody IdeaDTO ideaDto) throws Exception {
```

Comment:
Because of the global exception handler, I do not believe you need the throws exception. Instead, the exception you throw here should be a GlobalException, with an http status

Reply:
Noted.